### ISSUE TITLE
Adopt to breaking change in API of openDTU-onBattery greater 2024.02.09

### ISSUE THREAD

**Issue by waechtertoff** [henne49/dbus-opendtu#224]
**Describe the Problem**
The latest working version of openDTU-onBattery is  '2024.02.09'.
When using a younger version, current.log contains the message 'You do not have the latest OpenDTU Version to run this script,please upgrade your OpenDTU to at least version 4.4.3'.
Refer to [(https://github.com/hoylabs/OpenDTU-OnBattery/releases?page=2)].

**Device you are using**
1. Which Venus OS are you using? => v3.51
2. Which Victron Device? => Cerbo GX
3. Which DTU? => openDTU-onBattery (generic_esp32), version=2024.02.12 or later
4. Which Template Device?
=> Three Hoymiles inverters are connected to the openDTU box.

# 1st inverter
[INVERTER0]
Phase=3P
DeviceInstance=34
AcPosition=0

# 2nd inverter
[INVERTER1]
Phase=3P
DeviceInstance=35
AcPosition=0

# 3rd inverter
[INVERTER2]
Phase=L1
DeviceInstance=36
AcPosition=1

**Expected behavior**
The script should support different web API versions in parallel and neither force the user to stay at an older openDTU version nor to upgrade to a new version.

**Screenshots**
-

**Additional context**
-

[openDTU_livedata_status.json](https://github.com/user-attachments/files/17750233/openDTU_livedata_status.json)

**Comment by waechtertoff** [comment]
OK - will check again later that day and revert to you.

Kind regards,
Christoph

henne49 ***@***.***> schrieb am Do., 14. Nov. 2024, 17:06:

**Comment by waechtertoff** [comment]
Hi Henne,

I flashed 2024.06.03 again and retried. It did not work.

Then I checked the files with these results:
root@einstein:/data/dbus-opendtu-main# ls -al
-rw-r--r--    1 root     root         33000 Nov 14 15:02 dbus_service.py
-rw-r--r--    1 root     root            16 Nov 14 15:02 version.txt             /**< "1.2.40" */

root@einstein:/data/dbus-opendtu# ls -al
-rw-r--r--    1 root     root         31494 Jan  1  2024 dbus_service.py

I copied the commands from 'Installation' section to install the latest version. I wonder how the old version of the script survived.
The system was effectively running the old version.

When installing from directory /data/dbus-opendtu-main, I confirm that it is working with openDTU-onBattery 2024.06.03.

Sorry for interfering.

Kind regards,
Christoph
